src/test/scala/io/qbeast/spark/delta/DefaultFileIndexTest.scala

```diff
@@ -0,0 +1,128 @@
+package io.qbeast.spark.delta
```

Comment:
This one also seems to be missing a license header.

Reply:
Fixed on bfea6fc2